ai-gateway/app/routers/ingestion.py

```
@@ -0,0 +1,68 @@
+        if "full_text" in parsed_data:
+            del parsed_data["full_text"]
+            
+        return IngestionResponse(
+            file_id=file_id,
+            filename=file.filename,
+            content_type=file.content_type,
+            metadata=parsed_data,
+            llm_analysis=llm_analysis
+        )
+        
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)